source/installation/install-on-linux.txt

```diff
@@ -16,13 +16,12 @@ Install on Linux
 
 .. |arrow| unicode:: U+27A4
 .. |executable-name| replace:: ``mongosync``
-.. |mdb-download-link| replace:: `MongoDB Download Center <https://www.mongodb.com/try/download/enterprise?tck=docs_server>`__
 
 Overview
 --------
 
-Use this tutorial to manually install {+c2c-full-product-name+}
-{+version+} on Linux using a downloaded ``.tgz`` tarball.
+To configure {+c2c-full-product-name+} on Linux using a downloaded
+``.tgz`` tarball, follow the steps on this page.
```

Comment:
[suggestion]

Perhaps change this link so it looks the same as the Windows install page.